docs/2025/puzzles/day08.md

```diff
@@ -2,10 +2,123 @@ import Solver from "../../../../../website/src/components/Solver.js"
 
 # Day 8: Playground
 
+by [@mbovel](https://github.com/mbovel)
+
 ## Puzzle description
 
 https://adventofcode.com/2025/day/8
 
+## Data Model
+
+To solve this puzzle, we use a class to represent junction boxes:
+
+```scala
+/** A junction box in 3D space with an associated circuit ID. */
+class Box(val x: Long, val y: Long, val z: Long, var circuit: Int):
```

Comment:
Would it look more natural / beginner-friendly as

```scala
case class Box(x: Long, y: Long, z: Long, var circuit: Int)
```
I think it's fine either way, just a thought.